emuu-client/src/components/UserProfile/Profile.js

```diff
@@ -1,83 +1,49 @@
-                        </div>
-                    <div className="user-profile-img">
-                        <img id="prf-img" src="https://wallpaperaccess.com/full/170249.jpg" alt="" srcset=""/>
-                        </div>
-                        <div className={"userName"}>
-                            Moe
-                        </div>
-                         <p class="text-end">
-
-                                                                    <h4><div class = "p-5"> Subscribers: xxx </div></h4></p>
-                    </div>
-                        <div className="bottom-portion">
-
-                                                                    <p class="text-start">
-
-                                            <h2><div class = "p-4"> Uploaded Videos </div></h2>
-
-
-                                            <div className = "spacer">
-                                       <iframe width="382" height="215"
-                                         src="https://www.youtube.com/embed/UqjRknIf3oI"
-                                         title="YouTube video player" frameborder="0"
-                                       allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                                          allowfullscreen></iframe>
-                                        </div>
-
-                                       <div className = "spacer">
-                                  <iframe width="382" height="215"
-                                    src="https://www.youtube.com/embed/rDxv8jkYmb4"
-                                     title="YouTube video player" frameborder="0"
-                                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                                     allowfullscreen></iframe>
-                                 </div>
-                                <div className = "spacer">
-                                 <iframe width="382" height="215"
-                                   src="https://www.youtube.com/embed/0GBiA5JOht4"
-                                    title="YouTube video player" frameborder="0"
-                                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                                   allowfullscreen></iframe>
-                                 </div>
-                                 </p>
-
-
-
-
-
-                        </div>
-                        </div>
-            </div>
-
-
-
-
-    )
-
-
-
-
-
-
-
-
+  const [Banner, setBanner] = useState("");
+  const [ProfilePic, setProfilePic] = useState("");
+
+  getDoc(doc(db, "Users", displayName)).then((docSnap) => {
+    setBanner(docSnap.data().BannerUrl);
+    setProfilePic(docSnap.data().ProfilePictureUrl);
+  });
+
```

Comment:
Code looks much cleaner and easier to read after using prettier. Good job for getting that working!